sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -1486,17 +1506,31 @@
             return monoError(logger, new NullPointerException("'subscription' cannot be null."));
         }
 
+        final Context contextWithTrace = context.addData(AZ_TRACING_NAMESPACE_KEY, AZ_TRACING_NAMESPACE_VALUE);
+        final Context contextWithHeaders = contextWithTrace.addData(AZURE_REQUEST_HTTP_HEADERS_KEY, new HttpHeaders());
+        final String forwardToEntity = subscriptionOptions.getForwardTo();
```

Comment:
nit: addData is a fluentAPI, you can go:
```java
context = context.addData(AZ_TRACING_NAMESPACE_KEY, AZ_TRACING_NAMESPACE_VALUE)
    .addData(AZURE_REQUEST_HTTP_HEADERS_KEY, new HttpHeaders());
```

Reply:
done